website/addons/box/__init__.py

```diff
@@ -24,7 +24,7 @@
 ADDED_MANDATORY = []
 
 VIEWS = []
-CONFIGS = ['user', 'node']
+CONFIGS = ['accounts', 'node']
```

Comment:
```users``` config is a config related to one and only one user (i.e. twofactor), whereas ```accounts``` pertains only to external-account style addons that may be shared by multiple users.